portfolio/src/main/webapp/script2.js

```diff
@@ -0,0 +1,9 @@
+/* Javascript function that displays the current date and time in page */
```

Comment:
It would be useful for the reader if this script had a more descriptive name other than script2 so that it gives the reader an idea of what the script does.

Reply:
Same for the below scripts. Can all the scripts be merged into one file?